modules/csg/SCsub

```diff
@@ -6,7 +6,43 @@ Import("env_modules")
 
 env_csg = env_modules.Clone()
 
-# Godot source files
+env_csg.Append(CPPDEFINES=("MANIFOLD_PAR", "-1"))
+
+# Thirdparty source files
+
+thirdparty_obj = []
+
+thirdparty_dir = "#thirdparty/manifold/"
+thirdparty_sources = [
+    "src/cross_section/cross_section.cpp",
```

Comment:
The upstream CMakeLists.txt flags this as optional. Do we use it?
I don't see a reference to `cross_section.h` in our code nor in manifold code, so presumably `src/cross_section/` and `include/cross_section.h` can be removed.

Reply:
`include/meshIO.h` can also be removed as we're not including its `.cpp`.